pages/fellows/2024/JordanAshley.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: fellow
+pagetype: fellow
+shortname: JordanAshley
+permalink: /fellows/JordanAshley.html
+fellow-name: Jordan Science Ashley
+title: Jordan Science Ashley - IRIS-HEP Fellow
+active: True
+dates:
+  start: 2024-05-20
+  end: 2024-08-11
+photo: /assets/images/team/fellows-2024/Jordan-Ashley.png
+institution: University of Tennessee, Knoxville
+e-mail: jay.ashley@me.com
+project_title: Packaging jet substructure observable tools
+project_goal: >
+    The objective of this project is to repackage the EnergyFlow and Wasserstein tools for modern PyPI and conda-forge.
+mentors:
+  - Matthew Feickert (University of Wisconsin-Madison)
+  - Henry Schreiner (Princeton University)
+proposal: /assets/pdf/fellows-2024/proposal-jordan-ashley.pdf
```

Comment:
@rct225 just updated the Fellow new page docs (https://iris-hep.org/docs/fellow) but as this PR would beat the upload of all the Fellow proposal documents I'll have to defer to him on how the proposal should be named. I assume it will be something along the lines of

```
/assets/pdf/fellows-2024/IRIS001-proposal-Jordan-Ashley.pdf
```

but I'll need him to confirm.

Reply:
The name is fine for now.  There's more likely to be a collision by renaming it now since they will likely be bulk uploaded at some point.  I appreciate @j-s-ashley staying ahead of the game, so no need to change it.  Everything else looks OK.